### DISCUSSION TITLE
What is the Purpose of Callouts within Content Assets

### DISCUSSION THREAD

**Discussion by nicekiwi** [SalesforceCommerceCloud/sfcc-ci#404] [Team Posts]
What is the intended purpose for the "Callout" content area within Content assets? Our SFCC trainer was unable to answer.

**Comment by tsolon** [comment]
It is for whatever you want it to be.

It becomes the value of SlotContent.calloutMsg and is of type MarkupText.
It can then be rendered however your content slot rendering template uses
it.  The *sample* content slot rendering templates in both SFRA and SGJC
ignore it, but that does not mean you couldn't write your own.

The cartridge app_storefront_base does not use it for <isasset> so you
would need to write your own code if you wanted to render it as part of an
asset rendered on a page (apart from a content slot).


Tom Solon
Senior Technical Instructor | Salesforce
Mobile: +1 339-227-5327
/LinkedIn/tomsolon/ <https://www.linkedin.com/in/tomsolon/>


<http://smart.salesforce.com/sig/tsolon//us_mb/default/link.html>

**Comment by zlatinz** [comment]
One good usage for callouts is providing capability for custom header.
E.g. "Recommended products" vs "You may also like" vs. "Our best sellers" etc. above a content slot with product recommendations.
Also please make difference between content asset and content slot - callouts apply to content slots. Content Assets do not have such an attribute (although you may define such one, if you see fit).
